src/networking.c

```diff
@@ -2927,6 +2927,8 @@ void clientCommand(client *c) {
 "    Report tracking status for the current connection.",
 "NO-EVICT (ON|OFF)",
 "    Protect current client connection from eviction.",
+"STEALTHMODE (ON|OFF)",
```

Comment:
```suggestion
"STEALTH (ON|OFF)",
```
Honestly, this seems a little more short and to the point.